server.js

```diff
@@ -108,7 +108,8 @@ server.backend = function(base_dir, socket_emitter, user_config) {
     function discover() {
       //http://stackoverflow.com/a/24594123/1191579
       return fs.readdirSync(server_path).filter(function(p) {
-        return fs.statSync(path.join(server_path, p)).isDirectory();
+        //check if path exists, in case of broken symlink
+        return fs.existsSync(path.join(server_path, p)) && fs.statSync(path.join(server_path, p)).isDirectory();
```

Comment:
You're adding an extra "almost always true" condition here.  Maybe it would be better to just catch the return of fs.statSync before dereferencing it to call isDirectory() ?  Just not sure if the extra fs operation is worth the effort here.